src/OmniSharp.Abstractions/Models/v1/Completion/CompletionItem.cs

```diff
@@ -58,13 +59,21 @@ public class CompletionItem
         /// A string that should be inserted into a document when selecting
         /// this completion.When null or empty the label is used.
         /// </summary>
+        [Obsolete("Use TextEdit instead", error: true)]
         public string? InsertText { get; set; }
```

Comment:
I think you might as well break the contract and delete the property altogether. It's not populated anymore in the current implementation so it is not backwards compatible anymore anyway - therefore it would probably be more visible for the consumers that they need to change if it's not there at all